osidb/filters.py

```diff
@@ -795,3 +796,16 @@ class Meta:
             # versions fields
             "versions__version": ["exact"],
         }
+
+
+class AlertFilter(IncludeFieldsFilterSet, ExcludeFieldsFilterSet):
+    parent_uuid = CharFilter(field_name="object_id")
+    parent_model = CharFilter(field_name="content_type__model")
```

Comment:
I think this could be turned into `ChoiceFilter` and thus the whole query parameter annotation in the `osidb/api_views.py` might not be needed then but I have no strong preference and will leave it up to you.